### ISSUE TITLE
Bug in pycon lexer

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments#581]
(Original issue 874 created by mindprince on 2013-05-30T04:33:33.502457+00:00)

When using the following input with the pycon lexer, the output is not complete. Only the first line is shown in the output but the following lines are not shown.

```

>>> 1/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  ZeroDivisionError: integer division or modulo by zero

```

The output is http://pygments.org/demo/82880/

**Comment by Anteru** [comment]
(Original comment by tshatch on 2014-05-17T01:21:02.101075+00:00)

This is embarrassing.  I'll rework the console lexers -- anything that ends with a traceback will hit this.

**Comment by Anteru** [comment]
(Original comment by mindprince on 2014-05-17T05:09:22.040343+00:00)

Just a note, the following is how the python console prints tracebacks and it works correctly with pygments:
```
>>> 1/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: integer division or modulo by zero
```

I had mistakenly input the following:
```
>>> 1/0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  ZeroDivisionError: integer division or modulo by zero
```
which caused the issue.

It is still a bug as the output text doesn't match the input text. But it happens rarely, that's why you haven't seen more reports of this. :-)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-10-07T11:46:00.647912+00:00)

Closes #874: do not swallow incomplete tracebacks in the Python console lexer.

→ <<cset 5b840c2aca04>>
